package.json

```diff
@@ -43,9 +44,12 @@
     ]
   },
   "devDependencies": {
+    "@babel/core": "^7.6.4",
+    "@storybook/react": "^5.2.4",
```

Comment:
I think create react app already include . babel/core and babel-loader

Reply:
I know but this script installs the latest version which is necessary for storybook.js